### PR TITLE
TOMEE-2314 - Code cleanup in modules listed in description

### PR DESCRIPTION
**Pull request by cicekhayri**
This was little to big but next time will make JIRA ticket for each module  before start coding 

Modules worked in :

openejb-multicast,
openejb-axis,
openejb-common-cli,
openejb-ejbd,
openejb-server,
openejb-webservices,
tomee-common,

Usage of dimond operator, remove unused import and Java Doc

### CODE REVIEW

**Comment by jeanouii** [comment]
Would you please fix the conflict so I can merge it for you?
Yes next time, please prefer smaller PRs as they are easier to review

**Comment by cicekhayri** [comment]
Yes sorry about that, I will not rush and do it properly next time and will try to fix the conflicts today

**Comment by jeanouii** [comment]
Thank you. Lemme know when done, I'm happy to do the boilerplate under the cover to get this in.

**Comment by cicekhayri** [comment]
now i think the merge conflict is fixed

**Comment by jeanouii** [comment]
Here we go. Thanks a lot. Should be all in

**Comment by rzo1** [comment]
Hey @cicekhayri 

I am just curious: To which naming convention are you referring to in your first commit?

I thought, that constants consisting of more than one word will be separated by using "_", e.g. "CURRENTUNIVERSE" would be "CURRENT_UNIVERSE"

**Comment by cicekhayri** [comment]
Oh it should be like you wrote CURRENT_UNIVERSE, I can try to fix it later today

**Comment by rzo1** [comment]
Hi @cicekhayri

thanks for your fast response :) 

I think, that there are a few more constants following this pattern.

Shall I provide feedback, there I think, it could be improved via comments in the respective commit (https://github.com/apache/tomee/pull/231/commits/4e07471da871abc578c1c8a9c453c9be39dbe238) ?

**Comment by cicekhayri** [comment]
Absolutely it would be great 😊

File: server/openejb-multicast/src/main/java/org/apache/openejb/server/discovery/MulticastDiscoveryAgent.java [hunk 1]
```diff
@@ -46,7 +46,7 @@
  */
 public class MulticastDiscoveryAgent implements DiscoveryAgent, ServerService, SelfManaging {
 
-    private static final Logger log = Logger.getInstance(LogCategory.OPENEJB_SERVER.createChild("discovery").createChild("multicast"), MulticastDiscoveryAgent.class);
+    private static final Logger LOG = Logger.getInstance(LogCategory.OPENEJB_SERVER.createChild("discovery").createChild("multicast"), MulticastDiscoveryAgent.class);
```
Comment (rzo1) [inline server/openejb-multicast/src/main/java/org/apache/openejb/server/discovery/MulticastDiscoveryAgent.java]: Maybe we should change this to `LOGGER` to be consistent throughout the code-base?

File: server/openejb-webservices/src/main/java/org/apache/openejb/server/webservices/WsServlet.java [hunk 2]
```diff
@@ -43,7 +43,7 @@
     public static final String WEBSERVICE_CONTAINER = WsServlet.class.getName() + "@WebServiceContainer";
 
     private static final DefaultContext DEFAULT_CONTEXT = new DefaultContext();
-    private static final ThreadLocal<ServletEndpointContext> endpointContext = new ThreadLocal<ServletEndpointContext>();
+    private static final ThreadLocal<ServletEndpointContext> ENDPOINTCONTENT = new ThreadLocal<>();
```
Comment (rzo1) [inline server/openejb-webservices/src/main/java/org/apache/openejb/server/webservices/WsServlet.java]: `ENDPOINT_CONTEXT`

File: server/openejb-webservices/src/main/java/org/apache/openejb/server/webservices/saaj/SaajUniverse.java [hunk 3]
```diff
@@ -47,7 +47,7 @@ private static void setProperty(String name, String value) {
     public static final Type AXIS1 = Type.AXIS1;
     public static final Type AXIS2 = Type.AXIS2;
 
-    private static final ThreadLocal<LinkedList<Type>> currentUniverse =
+    private static final ThreadLocal<LinkedList<Type>> CURRENTUNIVERSE =
```
Comment (rzo1) [inline server/openejb-webservices/src/main/java/org/apache/openejb/server/webservices/saaj/SaajUniverse.java]: `CURRENT_UNIVERSE`

File: tomee/tomee-common/src/main/java/org/apache/tomee/installer/Installer.java [hunk 4]
```diff
@@ -32,31 +32,31 @@
     private Status status = Status.NONE;
     private boolean force;
 
-    private static final boolean listenerInstalled;
-    private static final boolean agentInstalled;
+    private static final boolean LISTENERINSTALLED;
```
Comment (rzo1) [inline tomee/tomee-common/src/main/java/org/apache/tomee/installer/Installer.java]: `LISTENER_INSTALLED`

File: tomee/tomee-common/src/main/java/org/apache/tomee/installer/Installer.java [hunk 5]
```diff
@@ -32,31 +32,31 @@
...
+    private static final boolean AGENTINSTALLED;
```
Comment (rzo1) [inline tomee/tomee-common/src/main/java/org/apache/tomee/installer/Installer.java]: `AGENT_INSTALLED`

**Comment by rzo1** [comment]
Hey @cicekhayri ,

I added related comments related to JNC. Maybe you can open a new JIRA for improvements and reference it in the next related PR :) - Thank you very much!

**Comment by cicekhayri** [comment]
Thank you so much I will try to fix it today 😊

**Pull request merged**